SKILL.tsv

```diff
@@ -4048,181 +4048,181 @@ SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRa
 SKILL_20150428_004048	{memo X}Effect applies #{CaptionRatio}# times
 SKILL_20150428_004049	{memo X}Back masking range100{nl}Cannot back mask for 10seconds in target area{nl}Use 1 Piece of Dimension
 SKILL_20150428_004050	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow breaks down to multiple pieces and bounces nearby.
-SKILL_20150428_004051	Defend range projectile #{CaptionRatio}# times{nl}Parvis Duration #{CaptionTime}#seconds
+SKILL_20150428_004051	$Blocks ranged projectiles: #{CaptionRatio}#{nl}Pavise Duration: #{CaptionTime}# seconds
 SKILL_20150428_004052	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 15 seconds{nl}Use 3 Ash tree
 SKILL_20150428_004053	{memo X}Attack #{SkillAtkAdd}#{nl}Splash #{CaptionRatio}#{nl}Use 1 Claymore
 SKILL_20150428_004054	{memo X}Attack #{SkillAtkAdd}#{nl}Casting 3 seconds{nl}Use 3 Pine Tree
 SKILL_20150428_004055	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Explode trap and magic circle to attack enemy.
 SKILL_20150428_004056	{memo X}Attack #{SkillAtkAdd}#{nl}#{CaptionRatio}# arrows{nl}Install range #{CaptionRatio2}#{nl}Use 2 wires
-SKILL_20150428_004057	Evasion -#{CaptionRatio}#{nl}Pointing AI duration 90 seconds
-SKILL_20150428_004058	Detect hiding monsters {nl}Hounding AI duration 90 seconds
-SKILL_20150428_004059	Use #{SpendPoison}# Poison Pot Poison
+SKILL_20150428_004057	$Evasion -#{CaptionRatio}#{nl}Pointing AI Duration: 90 seconds
+SKILL_20150428_004058	$Detects hidden monsters {nl}Hounding AI Duration: 90 seconds
+SKILL_20150428_004059	$Consumes #{SpendPoison}# Poison Pot Poison
```

Comment:
On a second thought, maybe slightly breaking the format and leaving it as "Consumes x Poison Pot(s)" is fine.